best-practices-cel/restrict-image-registries/restrict-image-registries.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-image-registries
+  annotations:
+    policies.kyverno.io/title: Restrict Image Registries in CEL expressions
+    policies.kyverno.io/category: Best Practices, EKS Best Practices in CEL 
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.11.0
+    kyverno.io/kubernetes-version: "1.26-1.27"
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Images from unknown, public registries can be of dubious quality and may not be
+      scanned and secured, representing a high degree of risk. Requiring use of known, approved
+      registries helps reduce threat exposure by ensuring image pulls only come from them. This
+      policy validates that container images only originate from the registry `eu.foo.io` or
+      `bar.io`. Use of this policy requires customization to define your allowable registries.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+  - name: validate-registries
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    validate:
+      cel:
+        expressions:
+          - expression: >-
```

Comment:
Could we use the variable `allContainers` that you already used in other policies?

Reply:
Sure. I have made the changes.